auto_dev/data/repo/templates/autonomy/scripts/run_single_agent.sh

```diff
@@ -0,0 +1,152 @@
+#! /bin/bash
+set -e
+
+if [[ "$#" -lt 1 ]]; then
+    echo "Usage: $0 <author/agent_name> [--force] [--debug]"
+    echo "  <author/agent_name>: Identifier for the agent (e.g., eightballer/donation_station)"
+    echo "  --force            : Remove existing build directory if it exists"
+    echo "  --debug            : Run the agent in DEBUG mode"
+    exit 1
+fi
+
+# Initialize variables
+FORCE=false
+DEBUG=false
+AGENT_IDENTIFIER=""
+
+# Parse arguments
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        --force)
```

Comment:
This only allows you to pass one argument?

Reply:
keeps looping as long as $# (number of args) are left

Reply:
col thanks